src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,24 @@
+import './Dropdown.css';
+
+import { ChangeEvent, FC } from 'react';
+
+interface DrodownProps {
+  name: string;
+  value: string;
+  options: ['All assets', 'Watchlist'];
```

Comment:
once again, you are setting a type that equals to an exact value. Are you sure it's the intention?

Reply:
You are right, changed it.

As explanation: I wanted to convert the project step by step to Typescript and initially only 'enlarge' a type on a per need basis. Now I noticed this approach is dumb, because it takes way longer :).